### PR TITLE
Fix increment / decrement in redis

### PR DESCRIPTION
**Pull request by exussum12**
also fixed a value incorrectly returned docblock says array but bool returned

### CODE REVIEW

File: system/libraries/Cache/drivers/Cache_redis.php [hunk 1]
```diff
@@ -294,7 +294,7 @@ public function get_metadata($key)
 			);
 		}
 
-		return FALSE;
+		return array();
```
Comment (narfbg) [inline system/libraries/Cache/drivers/Cache_redis.php]: Please revert this, it's supposed to return FALSE.

Comment (exussum12) [reply system/libraries/Cache/drivers/Cache_redis.php]: Update the doc block then ? It's currently saying it only returns array

Comment (narfbg) [reply system/libraries/Cache/drivers/Cache_redis.php]: Nope. I know it says only array, we simply don't bother with the possibility for `FALSE` being returned.

Comment (exussum12) [reply system/libraries/Cache/drivers/Cache_redis.php]: Ok. Seems odd. I've reverted this

**Pull request merged**